### PR TITLE
[coordinator] add graphite names to aggregation types

### PR DESCRIPTION
**Pull request by shreyassrivatsan**
**What this PR does / why we need it**:
In a previous change we added support for graphite ingestion through the prometheus remote write endpoints. As a part of this we added support to extend the graphite path with the aggregation type.. This PR makes sure that the aggregation type matches with the exact graphite names rather than the internal M3 aggregation types

**Special notes for your reviewer**:

**Does this PR introduce a user-facing and/or backwards incompatible change?**:
```release-note
The graphite aggregation option on mapping rules now writes metrics with graphite compatible paths, so will be different from prior to this release.
```

**Does this PR require updating code package or user-facing documentation?**:
```documentation-note
NONE
```

### CODE REVIEW

File: src/metrics/aggregation/type.go [hunk 1]
```diff
@@ -101,6 +101,30 @@ var (
 	}
 
 	typeStringMap map[string]Type
+
+	graphiteAggregationNames = map[Type][]byte{
```
Comment (shreyassrivatsan) [inline src/metrics/aggregation/type.go:105]: @benraskin92 does this mapping look ok?

**Review by jonathan-chronosphere — COMMENTED** [review]
All of this looks good.

**Review by benraskin92 — APPROVED** [review]
LGTM

**Review by sumeete — APPROVED** [review]

File: src/metrics/aggregation/types_options_test.go [hunk 2]
```diff
@@ -61,6 +61,7 @@ func TestOptionsSetDefaultTimerAggregationTypes(t *testing.T) {
 	o := NewTypesOptions().SetDefaultTimerAggregationTypes(aggTypes)
 	require.Equal(t, aggTypes, o.DefaultTimerAggregationTypes())
 	require.Equal(t, []float64{0.99, 0.9999}, o.Quantiles())
+	//fmt.Printf("type string %v\n", o.(*options).counterTypeStrings)
```
Comment (wesleyk) [inline src/metrics/aggregation/types_options_test.go:64]: rm debug line?

**Review by wesleyk — APPROVED** [review]

**Pull request merged**